app/src/main/java/ip/team13/petowner/ui/activities/ActivitiesFragment.kt

```diff
@@ -10,4 +20,65 @@ class ActivitiesFragment : BaseFragment() {
         get() = R.layout.activities_screen
 
     override val viewModel: ActivitiesViewModel by viewModel()
+
+    private lateinit var binding: ActivitiesScreenBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = DataBindingUtil.inflate(
+            layoutInflater,
+            R.layout.activities_screen,
+            container,
+            false
+        )
+        binding.lifecycleOwner = viewLifecycleOwner
+
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.adapter = ActivityAdapter(getMockData())
+        binding.viewModel = viewModel
+    }
+
+    // TODO remove mock data with real data from server
+    private fun getMockData() =
```

Comment:
Can we rather have a
`@get:Bindable
val mockedData: List<Activity> = arrayListOf(...)` field that we put inside the ViewModel? It is a bit weird to have the data held in the View layer.